DDG4/src/Celeritas.cpp

```diff
@@ -0,0 +1,87 @@
+#include <DDG4/Celeritas.h>
+
+#include <G4Threading.hh>
+#include <accel/AlongStepFactory.hh>
+#include <celeritas/field/UniformFieldData.hh>
+#include <celeritas/io/ImportData.hh>
+#include <accel/LocalTransporter.hh>
+#include <accel/SetupOptions.hh>
+#include <accel/SharedParams.hh>
+#include <accel/TrackingManagerOffload.hh>
+#include <G4Electron.hh>
+#include <G4Positron.hh>
+#include <G4Gamma.hh>
+
+#include <memory>
+
+using namespace celeritas;
+
+// Global shared setup options
+SetupOptions& CelerSetupOptions()
+{
+  static SetupOptions options = [] {
+    // Construct setup options the first time CelerSetupOptions is invoked
+    SetupOptions so;
+
+    // Set along-step factory
+    so.make_along_step = celeritas::UniformAlongStepFactory();
+    // NOTE: these numbers are appropriate for CPU execution
+    so.max_num_tracks = 1024;
+    // This will eventually go
+    so.max_num_events = 100000;
+    so.initializer_capacity = 1024 * 128;
+    // Celeritas does not support EmStandard MSC physics above 100 MeV
+    so.ignore_processes = {"CoulombScat"};
+
+    // Use Celeritas "hit processor" to call back to Geant4 SDs.
+    so.sd.enabled = false;
```

Comment:
We *definitely* shouldn't have this. Where were these lines copied from? We should probably use something closer to https://gitlab.cern.ch/bmorgan/athena/-/blob/6072d57711365aa3a58c38ff1134e3f0c569c78b/Simulation/G4Utilities/G4UserActions/src/Celeritas.cxx

Reply:
I think that's where I took it from. Am i looking in the wrong branch? https://gitlab.cern.ch/bmorgan/athena/-/blob/integrate-celeritas/Simulation/G4Utilities/G4UserActions/src/Celeritas.cxx?ref_type=heads&blame=1